src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -100,31 +100,31 @@
     <ProjectReference Include="..\..\NuGet.Core\NuGet.Common\NuGet.Common.csproj">
       <Ngen>True</Ngen>
       <NgenApplication>$(NgenApplicationDefault)</NgenApplication>
-      <NgenArchitecture>All</NgenArchitecture>
+      <NgenArchitecture>$(NgenArchitecture)</NgenArchitecture>
```

Comment:
What does not having this imply?

I see the recommendation in the original issue, but completely sure why.

Reply:
My recommendation was to delete it to use the default, this is basically equivalent of that because NgenAchitecture won't be set in most builds.